src/components/atoms/Tag/index.tsx

```diff
@@ -1,11 +1,11 @@
 import { FC } from "react";
 
 import * as S from "./styles";
-import { TagProps } from "./interface";
+import { TagStyleProps } from "./interface";
 
-const Tag: FC<TagProps> = ({ text, isDroppable = false }) => {
+const Tag: FC<TagStyleProps> = ({ text, isDroppable = false, fontSize }) => {
```

Comment:
Props with default should be always at the end, this way we can easily omit them, when we want to use defaults, just writing: `<Tag text="abs" fontSize="20px" />`
```suggestion
const Tag: FC<TagStyleProps> = ({ text, fontSize, isDroppable = false, }) => {
```

Reply:
Gonna fix it already 👯

Reply:
Actually does not matter, as this is not a function, but a functional component 😂